examples/cluster-ideal/cluster-ideal.yml

```diff
@@ -5,7 +5,7 @@
 #     N_c = 1 + 4 * pi * Rc**3 * N / (3 * L**3) = 1.22 in this case
 #
 temperature: 300
-random: {seed: fixed}
+random: {seed: hardware}
```

Comment:
Any requirements in terms of our examples and seed?

Reply:
I think the cluster move displaced too far as `dp=20` is exactly half the box lengtht. I have restored `fixed` and instead set `dp=19.9999` which passed. Otherwise we would randomly get mass center out of sync.